secretmanager/snippets/regional_samples/snippets_test.py

```diff
@@ -103,6 +112,27 @@ def secret_id(
         # Secret was already deleted, probably in the test
         print(f"Secret {secret_id} was not found.")
 
+@pytest.fixture()
+def regional_secret_version(
+    regional_client: secretmanager_v1.SecretManagerServiceClient,
+    regional_secret: Tuple[str, str, str, str],
+) -> Iterator[Tuple[str, str, str, str, str]]:
+    project_id, location_id, secret_id, _ = regional_secret
+
+    print(f"adding secret version to {secret_id}")
+    parent = f"projects/{project_id}/locations/{location_id}/secrets/{secret_id}"
+    payload = b"hello world!"
+    time.sleep(5)
+    version = regional_client.add_secret_version(
+        request={"parent": parent, "payload": {"data": payload}}
+    )
+
+    yield project_id, location_id, secret_id, version.name.rsplit("/", 1)[
+        -1
+    ], version.etag
+
+
+another_regional_secret_version = regional_secret_version
```

Comment:
question: is this used anywhere in the tests? I don't see it referenced in the code.

Reply:
Ah, this is used in one of the child branches of this PR. By mistake it is here. Removed from this PR.